www/js/services.js

```diff
@@ -281,10 +281,11 @@ angular.module('pvta.services', ['ngResource'])
   };
 })
 
-.service('Map', function($cordovaGeolocation){
+.factory('Map', function($cordovaGeolocation){
```

Comment:
I am not 100 percent sure what the difference is to be honest. I think I tried changing it because it was the only one that was a service and not a factory and I thought it might fix a bug I was having